examples/complete/variables.tf

```diff
@@ -6,10 +6,10 @@ variable "acl" {
 
 variable "grants" {
   type = list(object({
-    id          = string
-    type        = string
-    permissions = list(string)
-    uri         = string
+    id         = string
+    type       = string
+    permission = string
```

Comment:
can you explain this? looks like [docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#permissions) say its still a list

Reply:
@Benbentwo you're pointing to s3-bucket instead of the aws_s3_bucket_acl

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/version-4-upgrade#grant-argument

This is now a single string instead of a list. This is in the notes section too.